docs: Add notes about email confirmation for Spotify login (#20924)
supabase/supabase: apps/docs/pages/guides/auth/social-login/auth-spotify.mdx

## Add login code to your client app

> The following outlines the steps to sign-in using Spotify with Supabase Auth.

Reviewer: ```suggestion
The following outlines the steps to sign in using Spotify with Supabase Auth.
```

> 1. Call the sign-in method from the client library.

Reviewer: ```suggestion
1. Call the signin method from the client library.
```

`sign in` the verb, `signin` the noun (I guess adjective here 🤷🏼)

1. The user is redirected to the Spotify login page.

> 1. After completing the sign-in process, the user will be redirected to the your app with an error that says the email address needs to be confirmed. Simultaneously the user receives a confirmation email from Supabase Auth.

Reviewer: ```suggestion
1. After completing the sign-in process, the user will be redirected to your app with an error that says the email address needs to be confirmed. Simultaneously the user receives a confirmation email from Supabase Auth.
```

1. The user clicks the confirmation link in the email.

> 1. The user is brought back to the app and is not signed in.

Reviewer: `not` or `now`?

Author: Thanks, it should be `now`
```suggestion
1. The user is brought back to the app and is now signed in.
```
